packages/angular/projects/ui-angular/src/lib/primitives/button/__tests__/button.component.test.ts

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/angular';
+
+import { ButtonComponent } from '../button.component';
+
+describe('amplify-button', () => {
+  it('renders as expected', async () => {
+    const { container } = await render(
+      '<button amplify-button>My Button</button>',
+      {
+        declarations: [ButtonComponent],
+        componentProperties: { label: 'Password', fieldId: 'mockId' },
+      }
+    );
+    expect(container).toMatchSnapshot();
```

Comment:
Snapshot tests are a great start, but we should also verify that the button is rendered with the correct label
perhaps using `screen.getByText` query. Same comment for the other tests, we could add more assertions.

Reply:
Right, I 100% agree. However I wanted to scope this PR just for setting up `@testing-library/angular` (we have been blocked on this for a while). Then I will prepare another PR, across Angular and *Vue*, to establish those tests and also align any visual differences (e.g. #2875 that is WIP). 

Does that sound good to you?